tools/wptrunner/wptrunner/tests/browsers/test_sauce.py

```diff
@@ -95,7 +96,8 @@ def test_sauceconnect_cleanup():
 
         sleep.assert_called()
 
-
+@pytest.mark.xfail(sys.version_info >= (3,),
+                   reason="importing wptrunner.browser.sauce throws ImportError on Py3")
```

Comment:
This isn't true any more (hence why this fails; as otherwise it would be skipped from line 10), maybe just:

```suggestion
                   reason="fails on Py3")
```

Reply:
Code updated. Thanks!